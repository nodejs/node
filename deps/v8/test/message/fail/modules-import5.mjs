// Copyright 2016 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import {a as b} from "modules-import5.mjs";
export {c as a} from "modules-import5.mjs";
import {c} from "modules-import5.mjs";
