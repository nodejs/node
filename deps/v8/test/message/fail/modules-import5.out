*%(basename)s:6: SyntaxError: The requested module 'modules-import5.mjs' does not provide an export named 'c'
export {c as a} from "modules-import5.mjs";
        ^^^^^^
SyntaxError: The requested module 'modules-import5.mjs' does not provide an export named 'c'

